content_scripts/link_hints.coffee

```diff
@@ -152,6 +152,7 @@ LinkHints =
       when "hover" then mode = HOVER_LINK
       when "focus" then mode = FOCUS_LINK
     #
+    mode.options = registryEntry?.options
```

Comment:
Out of curiosity, why don't I need to make a clone of `mode` here? I thought it would modify the original `DOWNLOAD_LINK_URL` hence resulted in side-effects to subsequent calls with unassigned options but this worked just alright lol...

Reply:
Ah, nvm, dumb question... Just realised that I was looking at the wrong line in debugger. It does change the original mode, but `mode.options` also gets overwritten every time the function is called.